commit 76945312a9d2e786bebc9bb5711f5962365c5c20
Author: Joe Barnett <jbarnett@stanfordalumni.org>
Date:   Thu Apr 21 15:57:16 2016 -0700

    Add dkms.conf and tweak Makefile to work with dkms
    
    Specifically, allow building for an installed kernel that is not the running kernel

diff --git a/Makefile b/Makefile
index 0e4b2a8..0ddf666 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,10 @@
 facetimehd-objs := fthd_ddr.o fthd_hw.o fthd_drv.o fthd_ringbuf.o fthd_isp.o fthd_v4l2.o fthd_buffer.o fthd_debugfs.o
 obj-m := facetimehd.o
 
+KVERSION := $(KERNELRELEASE)
+ifeq ($(origin KERNELRELEASE), undefined)
 KVERSION := $(shell uname -r)
+endif
 KDIR := /lib/modules/$(KVERSION)/build
 PWD := $(shell pwd)
 
